lib/fetching/content-services/twitter-content-service.js

```diff
@@ -80,11 +80,24 @@ TwitterContentService.prototype._getStream = function() {
 
 TwitterContentService.prototype._parse = function(data) {
   var author = data.user ? data.user.screen_name : null;
+  var metadata = {
+    replyTo: data.in_reply_to_status_id_str ? data.in_reply_to_status_id_str : '',
+    location: data.user.location ? data.user.location : '',
+    verified: data.user.verified ? data.user.verified : false,
+    followerCount: data.user.followers_count ? data.user.followers_count : 0,
+    friendCount: data.user.friends_count ? data.user.friends_count : 0,
+    geoEnabled: data.user.geo_enabled ? data.user.geo_enabled : false,
+    latitude: data.coordinates ? data.coordinates[0] : 0,
+    longitude: data.coordinates ? data.coordinates[1] : 0,
+    retweetCount: data.retweet_count ? data.retweet_count : 0,
+    favouriteCount: data.favorite_count ? data.favorite_count : 0
+  };
   return {
     authoredAt: new Date(data.created_at),
     fetchedAt: new Date(),
     content: data.text,
     author: author,
+    metadata: metadata,
```

Comment:
Codeclimate caught the one in the facebook content service, and I've fixed that. :)